spec/unit/resource_helpers_spec.rb

```diff
@@ -0,0 +1,87 @@
+require_relative 'spec_helper'
+include CernerSplunk::PathHelpers
```

Comment:
I don't think it's possible. Running individual specs fails in any case after doing this. It doesn't seem to mix-in the code in the right place.